apps/server/src/modules/classes/entity/class.entity.spec.ts

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable no-new */
+import { setupEntities } from '@shared/testing';
+import { classEntityFactory } from '@shared/testing/factory/class.factory';
+import { ObjectId } from '@mikro-orm/mongodb';
+import { ClassEntity } from './class.entity';
+
+describe('Class Entity', () => {
```

Comment:
You can do just _ClassEntity.name_ so if the class name changes, this string will also be correct without requiring additional refactor.